external/PATCH-01-STRATUM

```diff
@@ -2928,17 +2933,19 @@ index 9f4bb3a9..e0154d94 100644
 +       goto internal_error;
 +    }
 +    tdi_status = table->entryGetFirst(
-+        *tdirt_session, tdi_dev_target,
++        *tdirt_session, *dev_tgt,
 +        *flags, table_key.get(),
 +        table_data.get());
++
 +    if (tdi_status != TDI_SUCCESS) {
 +       goto internal_error;
 +    }
  
      table_keys->push_back(std::move(table_key));
      table_datums->push_back(std::move(table_data));
-@@ -523,18 +563,27 @@ template <typename T>
- 
+   }
+   if (entries == 1) return ::util::OkStatus();
```

Comment:
What does entry ==1 mean here?

Reply:
its the existing code, just the line number changed, its checking for first table entry and then trys to fetch other entries if any